src/operation/aggregate/test.rs

```diff
@@ -287,7 +287,7 @@ async fn handle_write_concern_error() {
     let response = CommandResponse::with_document(doc! {
         "cursor" : {
             "firstBatch" : [ ],
-            "id" : 0 as i64,
+            "id" : 0_i64,
```

Comment:
I knew that you could suffix a literal with the integer type, but I never thought of using one of the underscores that gets ignored to make it more readable before!